x-pack/test/functional/apps/ingest_pipelines/ingest_pipelines.ts

```diff
@@ -18,10 +41,26 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
     });
 
     it('Loads the app', async () => {
-      await log.debug('Checking for section heading to say Ingest Node Pipelines.');
+      log.debug('Checking for section heading to say Ingest Node Pipelines.');
 
       const headingText = await pageObjects.ingestPipelines.sectionHeadingText();
       expect(headingText).to.be('Ingest Node Pipelines');
     });
+
+    it('Creates a pipeline', async () => {
+      await pageObjects.ingestPipelines.createNewPipeline(PIPELINE);
+
+      const pipelinesList = await pageObjects.ingestPipelines.getPipelinesList();
+      const newPipelineExists = Boolean(
+        pipelinesList.find(pipelineName => pipelineName === PIPELINE.name)
```

Comment:
I wanted to know if this `pipelinesList.find()` needed to be async or if this is some kind of ES6 JS find. I am not the best at JS but that was my only question.

Reply:
This is just using [Array.find()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) so should be OK as is.